update reserved keywords in log payload for telemetry docs (#15133)
kyma-project/kyma: docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

- Kubernetes Filter
- Rewrite_Tag Filter

### Reserved log attributes

> The log attribute with name `kubernetes` is a special attribute being enriched by the `kubernetes` filter. When using that attribute as part of your structured log payload, the metadata enriched by the filter gets overwritten by the payload data. Filters which are relying on that metadata might not work anymore as expected.

Reviewer: ```suggestion
The log attribute named `kubernetes` is a special attribute that's enriched by the `kubernetes` filter. When you use that attribute as part of your structured log payload, the metadata enriched by the filter are overwritten by the payload data. Filters that rely on the original metadata might no longer work as expected.
```

> Furthermore, the prefix `__k8s__` is used for internals. When using the attribute prefix in your log data, the data might get overwritten.

Reviewer: "internal" what? Internal use? Internal users?

```suggestion
Furthermore, the prefix `__k8s__` is used for internals. When you use the attribute prefix in your log data, the data might be overwritten.
```

no "get" passive in our docs.

- Kubernetes Filter
- Rewrite_Tag Filter

> In addition, the strings `__k8s__` and `kubernetes` are reserved by the LogPipeline and should not be used in the log payload, otherwise they will be overwritten.

Reviewer 2: ```suggestion
### Reserved log attributes
The log attribute with name `kubernetes` is a special attribute being enriched by the `kubernetes` filter. When using that attribute as part of your structured log payload, the metadata enriched by the filter gets overwritten by the payload data. Filters which are relying on that metadata might not work anymore as expected.

Furthermore, the prefix `__k8s__` is used for internals. When using the attribute prefix in your log data, the data might get overwritten.
```
